string collisionObjectId
---
bool result
string errorCode
